Source/RMG/UserInterface/Widget/RomBrowserWidget.cpp

```diff
@@ -479,8 +479,19 @@ void RomBrowserWidget::customContextMenuRequested(QPoint position)
         this->action_PlayGameWith->setText("Play Game with Disk");
     }
 
+    if (view == this->listViewWidget)
+    { // list view
+        this->categorySubmenu = this->mainContextMenu->addMenu(tr("&Show/Hide Columns"));
```

Comment:
1) why the `&`
2) please remove the `tr` usage, RMG wont support translations anytime soon